joplin/templates/wagtailadmin/base.html

```diff
@@ -22,10 +21,26 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 {% include 'session_security/all.html' %}
+{% comment %}
+maybe there is a better way/place to pull in this script from the preflight app
```

Comment:
Yes, this could be rendered as a webpack bundle like the other js files. Let's make our js loading consistent.